src/App.svelte

```diff
@@ -198,11 +214,47 @@
     background-color: rgb(0, 157, 54);
   }
 
-  .refresh {
+  button.icon {
     padding: 0.8rem 1rem;
   }
+
   .date {
     text-align: center;
     margin-top: 0;
   }
+
+  /* some phones are smaller than others */
```

Comment:
This comment does not say much about the following code. Maybe we could remove it? Same with the comment above the media query.